capstone.html

```diff
@@ -0,0 +1,153 @@
+                    <li>
+                        <a href="#" class="is-actives">Join</a>
+                    </li>
+                    <li>
+                        <a href="#" class="is-actives">Sponser</a>
+                    </li>
+                    <li>
+                        <a href="#" class="is-actives">News</a>
+                    </li>
+                </ul>
+            </div>
+            <div class="company">
+                <button>EEP Taining</button>
+            </div>
+
+        </div>
+    </nav>
+</header>
+    <div class="main">
+        <div class="hello" id="home">
+            <p class="heading">"Hello! Engineers in the world"</p>
+            <h2>Welcome to Electrical <br> enginering portal  launches <br>on Fri,july 2022</h2>
+            <p class="introduction">Power engineering, also called power systems engineering, is a subfield of electrical engineering that deals with the generation, transmission, distribution, and utilization of electric power, and the electrical apparatus connected to such systems.this seminar is a gate way  to power engineering.</p>
+            <h4>2022.7.29(FRI)~5(THU)</h4>
+            <p class="hall"><span>&#64;</span>Addis Ababa institute of technology,center of research and technology</p>
+        </div>
+        <div class="main-catagories" id="about">
+            <h2 class="under-line">main catagories</h2>
+            <div class="main-catagories-inner">
+                <div class="substation-design">
+                    <div class="images">
+                        <img src="images/design2.png" alt="">
+                    </div>
+                    <div class="titles">
+                        <h3>substation design</h3>
+                    </div>
+                    <div class="introductions">
+                        <p>Building a new substation or retrofitting the old one is a complex process full of design and engineering tasks to be worked on. The main steps in substation design and engineering are as follows</p>
+                    </div>
+                    
+                </div>
+                <div class="substation-design">
+                    <div class="images">
+                        <img src="images/tr-line.jpg" alt="">
+                    </div>
+                    <div class="titles">
+                        <h3>substation construction</h3>
+                    </div>
+                    <div class="introductions">
+                        <p> Construction of a new substation requires a large amount of materials, equipment and labour. You may hear or notice some construction activities depending on your proximity to the substation site.</p>
+                    </div>              
+                </div>
+                <div class="substation-design">
+                    <div class="images">
+                        <img src="images/wind-power.png" alt="">
+                    </div>
+                    <div class="titles">
```

Comment:
- [ ] Kindy fix your shrinking images, it's not making your UI look great 👍 . I think you should consider using `object-fit: cover `on the image styling.

![image](https://user-images.githubusercontent.com/63473584/182240883-080e93e2-db36-4b60-a1f3-8a6659c66798.png)